docs/pages/messages/missing-root-task-in-turbo-json.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Missing root task in turbo.json
+description: Missing root task in turbo.json error
+---
+
+# Missing root task in turbo.json
+
+## Why This Error Occurred
+
+We call the scripts defined in the monorepo's root `package.json` the root tasks. These tasks often call `turbo`
+themselves. For example:
+
+```json package.json
+{
+  "scripts": {
+    "build": "turbo run build"
+  }
+}
+```
+
+This creates a problem when we declare task dependencies with `^`. A quick recap, `^<task>` means all of the `<task>`
+entries in the package's dependencies, e.g. if package `a` depends on packages `b` and `c`, and `a#build` depends on
+`^build`, then its dependencies will be `b#build` and `c#build`. We call these topological dependencies. Topological
+dependencies work well for many common situations, such as when you want to build your dependencies before building
+your own package.
+
+```json turbo.json
+{
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build"]
+    }
+  }
+}
+```
+
+However, following package dependency relations is slightly problematic because the root package is an implicit dependency
```

Comment:
Let's keep it a little more terse here in general.

"However, following package dependencies can be problematic because the root package is a part of the dependency graph for all package inside your workspace per JavaScript workspace conventions. Therefore, we don't execute root tasks by default for convenience so you can still write "turbo run build" in your root "build" script."

That should be closer but feel free to edit if there's more clarity to be had (keeping brevity in mind).

Reply:
Hmm, "part of the dependency graph" is a little confusing imo. Maybe "the root package is always a dependency for all packages inside your workspace"